src/workerd/jsg/resource.h

```diff
@@ -759,6 +759,13 @@ struct ResourceTypeBuilder {
                             v8::PropertyAttribute::ReadOnly | v8::PropertyAttribute::DontEnum));
   }
 
+  template<typename T>
+  inline void registerReadonlyInstanceProperty(const char* name, T value) {
```

Comment:
Why `const char*` and not `kj::StringPtr`? Probably more efficient as `kj::StringPtr` to avoid redundant `strlen()` calls.